src/components/Colors/useColorContext.js

```diff
@@ -13,7 +13,9 @@ export const useColorContext = () => {
     // precomputed css rules which are often used
     const colorRules = {
       textColor: css({
-        color: colorScheme.text
+        color: colorScheme.rawColors
+          ? [colorScheme.rawColors.text, colorScheme.text]
+          : colorScheme.text
```

Comment:
I am struggling to understand why we need this switch. Shouldn't the context itself already take care of return the right thing? (See above)